components/form/LabeledSelect.vue

```diff
@@ -329,7 +333,9 @@ export default {
   }
 
   .labeled-container {
-    padding: $input-padding-sm 0 0 $input-padding-sm;
+    // Make LabeledSelect and LabeledInput the same height so they
+    // don't wiggle when you toggle between them.
+    padding: 7px 0 0 $input-padding-sm;
```

Comment:
I think we should try to identify a way to use only variable over magic numbers. As I did not entirely understand what element and part has this very size, even with the comment, it may be convenient straight with a variable.